From 9c9cb69397dad29f7874d2ff451528f01ec783cf Mon Sep 17 00:00:00 2001
From: Sveinung Kvilhaugsvik <sveinung84@users.sourceforge.net>
Date: Tue, 23 May 2017 15:45:56 +0200
Subject: [PATCH 2/2] webperimental: reduce Manhattan Project power.

The Manhattan Project allows its owner's Spy units to perform the action
"Suitcase Nuke" or, depending on veteran level, "Suitcase Nuke Escape". It
was made slightly less powerful in hrm Feature #661962 when informants (non
veterans) and handlers (the first veteran level) lost the ability to escape
after hiding the nuke in a city. Make it even less powerful by reducing the
odds of successfully hiding a nuclear device.

See hrm Feature #662938
---
 data/webperimental/README.webperimental |  1 +
 data/webperimental/effects.ruleset      | 40 +++++++++++++++++++++++++++++++++
 2 files changed, 41 insertions(+)

diff --git a/data/webperimental/README.webperimental b/data/webperimental/README.webperimental
index 13c9117..ffc112b 100644
--- a/data/webperimental/README.webperimental
+++ b/data/webperimental/README.webperimental
@@ -16,6 +16,7 @@ Differences compared to the classic ruleset are:
   - unit stack blocks bribe/sabotage unit needs killstack.
 - Spies can steal parts of other player's world map.
 - Spies belonging to the owner of the Manhattan Project can nuke cities.
+  - a higher veteran level gives a higher chance of success.
   - agents and spymasters may even escape before the nuke detonates.
 - Explorers can establish embassies once Writing is discovered.
   This action consumes the explorer.
diff --git a/data/webperimental/effects.ruleset b/data/webperimental/effects.ruleset
index 54d5380..8229301 100644
--- a/data/webperimental/effects.ruleset
+++ b/data/webperimental/effects.ruleset
@@ -2426,6 +2426,46 @@ reqs	=
       "UnitFlag", "Cities", "Local"
     }
 
+; informant (veteran level 0) and handler (veteran level 1) does the
+; non escape version
+[effect_suitcase_nuke_odds_base]
+type	= "Action_Odds_Pct"
+value	= -69
+reqs	=
+    { "type",   "name",          "range"
+      "Action", "Suitcase Nuke", "Local"
+    }
+
+; give handler (veteran leven 2) better odds
+[effect_suitcase_nuke_odds_first_veteran]
+type	= "Action_Odds_Pct"
+value	= 6
+reqs	=
+    { "type",       "name",          "range"
+      "Action",     "Suitcase Nuke", "Local"
+      "MinVeteran", "1",             "Local"
+    }
+
+; agent (veteran level 2) and spymaster (veteran level 3) does the
+; escape version
+[effect_suitcase_nuke_esc_odds_base]
+type	= "Action_Odds_Pct"
+value	= -57
+reqs	=
+    { "type",   "name",                 "range"
+      "Action", "Suitcase Nuke Escape", "Local"
+    }
+
+; give spymaster (veteran leven 3) better odds
+[effect_suitcase_nuke_esc_odds_last_veteran]
+type	= "Action_Odds_Pct"
+value	= 6
+reqs	=
+    { "type",       "name",                 "range"
+      "Action",     "Suitcase Nuke Escape", "Local"
+      "MinVeteran", "3",                    "Local"
+    }
+
 [effect_steal_gold_base]
 type    = "Max_Stolen_Gold_Pm"
 value   = 50
-- 
2.1.4

